_includes/google-search.html

```diff
@@ -0,0 +1,12 @@
+<script>
+    (function() {
+      var cx = 'e20a3e7ce37c2454c';
+      var gcse = document.createElement('script');
+      gcse.type = 'text/javascript';
+      gcse.async = true;
+      gcse.src = 'https://cse.google.com/cse.js?cx=' + cx;
+      var s = document.getElementsByTagName('script')[0];
```

Comment:
I wonder if we can avoid enumerating all `script` nodes?  E.g. by giving the `script` node we want an `id` attribute and using `getElementById`, or by querying only the first `script` node using `querySelector('script')`?

By the way, which `script` node is this looking for?  The one that contains this code?

Reply:
I really just took this from a tutorial and it seems to work so... I'm not sure about modifying it :) 
https://jekyllcodex.org/without-plugin/search-google/